app/views/layouts/application.html.erb

```diff
@@ -8,6 +8,12 @@
     <script src="https://kit.fontawesome.com/8c1b4cb35e.js" crossorigin="anonymous"></script>
     <%= stylesheet_link_tag "application", "data-turbo-track": "reload" %>
     <%= javascript_include_tag "application", "data-turbo-track": "reload", defer: true %>
+
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+    <!-- Calling the minified stylesheet from the Bootstrap library-->
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@4.5.3/dist/css/bootstrap.min.css" rel="stylesheet">
+    <!-- Calling the minified stylesheet from the Datepicker extension -->
+    <link rel="stylesheet"  href="https://cdnjs.cloudflare.com/ajax/libs/bootstrap-datepicker/1.9.0/css/bootstrap-datepicker.min.css">
```

Comment:
You forgot to remove lines 12 to 16.  I request a change to this.

Reply:
Done